transmogrifier/sources/xml/dspace_mets.py

```diff
@@ -151,6 +151,12 @@ def get_optional_fields(self, xml: Tag) -> dict:
         for publisher in xml.find_all("mods:publisher", string=True):
             fields.setdefault("publication_information", []).append(publisher.string)
 
+        # publishers
```

Comment:
Similar comment to above. How do we know if a source has multiple elements tagged with `mods:publisher`? 🤔

Reply:
We wouldn't know but this transform is set up handle multiple publisher elements